### DISCUSSION TITLE
How to troubleshoot high CPU on sensor?

### DISCUSSION THREAD

**Discussion by lonelyadmin** [Security-Onion-Solutions/securityonion#4627] [Unsupported Versions]
so-status shows all OK except for so-filebeat, so-strelka-* except coordinator in starting state. High CPU is being shown by top >95% for `salt-minion` and `/usr/sbin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock` processes. `docker stats` shows single digit CPU usage.

How can I drilldown into what is going on?

**Comment by weslambert** [comment]
You can use Grafana to get a better idea of CPU utilization for certain processes (like ES/Logstash).  You can also try tailing the minion log in `/opt/so/log/salt/minion`  What are the specs of the box?
